<launch>
  <!-- Node launching -->
  <node pkg="mercator_sensor_fusion_ros_pkg" type="calibration_cam_lidar" name="calibration_cam_lidar" output="screen">
    <!-- Parameters -->
    <param name="collect_time" value="60.0" />
    <param name="data_max_size" value="50" />
    
    <!-- Topic remappings -->
    <remap from="cam_poses" to="/input_cam_poses" />
    <remap from="lidar_poses" to="/input_lidar_poses" />
    <remap from="ground_truth_poses" to="/input_ground_truth_poses" />
  </node>
</launch>
